internal/k8sCommon/k8sclient/endpoint_test.go

```diff
@@ -4,13 +4,13 @@
 package k8sclient
 
 import (
+	"gotest.tools/v3/assert"
```

Comment:
Should probably change this to `"github.com/stretchr/testify/assert"`

Reply:
"github.com/stretchr/testify/assert" does not support the DeepEqual() used in this test, probably used to support that but not anymore after the version bump.